control/src/CtControl.cpp

```diff
@@ -632,10 +632,14 @@ void CtControl::getStatus(Status& status) const
   HwInterface::Status aHwStatus;
   m_hw->getStatus(aHwStatus);
   DEB_TRACE() << DEB_VAR1(aHwStatus);
-  if(aHwStatus.acq == AcqFault)
+  if(aHwStatus.acq == AcqFault) {
     status.AcquisitionStatus = AcqFault;
-  else if(status.AcquisitionStatus == AcqReady)
+  } else if (aHwStatus.acq == AcqReady) {
     status.AcquisitionStatus = aHwStatus.acq;
+  } else if(status.AcquisitionStatus == AcqReady) {
+     status.AcquisitionStatus = aHwStatus.acq;
+  }
+  DEB_TRACE() << DEB_VAR1(status);
 }
 
```

Comment:
The global acquisition status will be at AcqReady as soon as the camera is ready. This is wrong because we shouldn't
return AcqReady if there is still some active Processing tasks.
For me the previous code is correct, what was wrong with this?

Reply:
This can also be reverted